lib/helpers/time.js

```diff
@@ -53,6 +85,19 @@ function getDisplayFormat(time) {
   return moment.tz(time, timezone).format('h:mm a z');
 }
 
+function getDisplayFormatForDays(days) {
+  if(days.length === 5) {
+    return 'all weekdays';
+  }
+
+  return days.reduce((previous, current, index) => {
+    if(index == days.length - 1 && days.length > 1) {
+      return `${previous} and ${current}`;
+    }
+    return `${previous}${previous.length > 0 ? ', ' : ''}${current}`;
+  }, '');
```

Comment:
Oops! Yeah, for some reason I thought that array syntax was new in ES6. It's not, coffeescript is just coming back to haunt me :ghost: